draft-ietf-tls-dtls-connection-id.md

```diff
@@ -366,7 +369,10 @@ in changing a peer address if they are able to rewrite source addresses
 and if replayed packets are able to arrive before any original. 
 
 - There is a strategy for ensuring that the new peer address is able to 
-receive and process DTLS records. No such test is defined in this	specification. 
+receive and process DTLS records. No such test is defined in this specification.
+
+If any of the above conditions fail, the receiver MUST silently discard the
+received datagram.
```

Comment:
OK, so there are two different statements here:

1. Don't update the IP and send
2. Don't even accept the packet.

If we want (2) then we can simplify, right?

Reply:
Therefore make general case of RFC6347

> In general, DTLS implementations SHOULD silently discard records with bad MACs or that are otherwise invalid.

mandatory and remove the option to alert.

Reply:
I removed the sentence being discussed here.